src/main/java/Duke.java

```
@@ -1,10 +1,86 @@
+                System.out.println(dividingLine + "OK, I've marked this task as not done yet:");
+                System.out.println("[" + tasks[unmarkNum].getStatusIcon() + "] " + tasks[unmarkNum].description
+                        + dividingLine);
+            }
+            else if(line.contains("todo")){
+                line = line.replace("todo ", "");
+                addTask(new Todo(line));
+                System.out.println(dividingLine + "Got it. I've added this task:");
+                System.out.println(tasks[taskCount].toString());
+                System.out.println("Now you have " + (taskCount + 1) + " tasks in the list." + dividingLine);
+                taskCount++;
+            }
+            else if(line.contains("deadline")){
+                line = line.replace("deadline ", "");
+                addTask(new Deadline(line.substring(0, line.indexOf(" /") - 1),
+                        line.substring(line.indexOf("/by ") + 4)));
+                System.out.println(dividingLine + "Got it. I've added this task:");
+                System.out.println(tasks[taskCount].toString());
+                System.out.println("Now you have " + (taskCount + 1) + " tasks in the list." + dividingLine);
+                taskCount++;
+            }
+            else if(line.contains("event")){
+                line = line.replace("event ", "");
+                addTask(new Event(line.substring(0, line.indexOf(" /") - 1),
+                        line.substring(line.indexOf("/from ") + 6, line.indexOf(" /to",
+                                line.indexOf("/from") - 1)),
+                        line.substring(line.indexOf("/to ") + 4)));
+                System.out.println(dividingLine + "Got it. I've added this task:");
+                System.out.println(tasks[taskCount].toString());
+                System.out.println("Now you have " + (taskCount + 1) +" tasks in the list." + dividingLine);
+                taskCount++;
+            }
+            else{
```

This condition should throw an exception instead of showing that the task has been added